package.json

```diff
@@ -42,5 +42,8 @@
     "prettier": "^2.5.1",
     "ts-node": "^10.5.0",
     "typescript": "^4.5.5"
+  },
+  "dependencies": {
+    "react-number-format": "^4.9.3"
```

Comment:
Mamy 3 (w sumie 4 bo jeszcze SDK) `package.json`:
- `package.json` - wspólne pakiety
- `packages/client/package.json` - pakiety używane tylko przez klienta
- `packages/server/package.json` - pakiety używane tylko przez serwer

To jest typowo biblioteka do UI także lepiej dodać do pakietów klienta.

```bash
yarn remove react-number-format && cd packages/client && yarn add react-number-format
```